src/test/kotlin/week1/step3/CalculatorTest.kt

```diff
@@ -0,0 +1,23 @@
+package week1.step3
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class CalculatorTest {
```

Comment:
클래스 이름은 `CarTest`가 적절해 보여요.

```suggestion
class CarTest {
```

Reply:
intellij 가 이상해서 테스트 파일을 만들면 테스트가 동작되지 않아 기존 파일을 복붙하여 발생한 실수입니다.:sob:
수정하였습니다.
[608c59a805c9bf46115f0c12175733a70e7b3861]